addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -618,17 +618,22 @@ export class X2ManyFieldDialog extends Component {
             this.archInfo.arch = this.archInfo.xmlDoc.outerHTML;
         }
 
-        const { autofocusFieldId, disableAutofocus } = this.archInfo;
+        const { autofocusFieldIds, disableAutofocus } = this.archInfo;
         if (!disableAutofocus) {
             // to simplify
             useEffect(
                 (isInEdition) => {
                     let elementToFocus;
                     if (isInEdition) {
-                        elementToFocus =
-                            (autofocusFieldId &&
-                                this.modalRef.el.querySelector(`#${autofocusFieldId}`)) ||
-                            this.modalRef.el.querySelector(".o_field_widget input");
+                        for (const id of autofocusFieldIds) {
+                            elementToFocus = this.modalRef.el.querySelector(`#${id}`);
+                            if (elementToFocus) {
+                                break;
+                            };
+                        };
```

Comment:
```suggestion
                        for (const id of autofocusFieldIds) {
                            elementToFocus = this.modalRef.el.querySelector(`#${id}`);
                            if (elementToFocus) break;
                        }
```
Purely cosmetic, feel free to ignore :blush:

Reply:
Yep, ignore. We have a linter, and brackets are enforced.

Reply:
Oh OK, sorry, didn't know :sweat_smile: 
Thanks for the feedback!